baseapp_auth/graphql/filters.py

```
@@ -17,5 +18,8 @@ class Meta:
         model = get_user_model()
         fields = ["q", "order_by"]
 
-    def filtesearchr_q(self, queryset, name, value):
-        return queryset.filter(Q(first_name__icontains=value) | Q(last_name__icontains=value))
+    def filter_by_q(self, queryset, name, value):
```

Ruff (0.14.14)

[warning] Unused method argument: `name`

(ARG002)